### PR TITLE
Add plausible

### PR DESCRIPTION
**Pull request by carletex**
Adding [plausible](https://plausible.io/) to the project (as in v1)

### CODE REVIEW

File: packages/nextjs/next.config.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 // @ts-check
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { withPlausibleProxy } = require("next-plausible");
```
Comment (carletex) [inline packages/nextjs/next.config.js:3]: :(

image

Comment (technophile-04) [reply packages/nextjs/next.config.js:3]: Ohh yes encountered this earlier, to handle it we can convert `next.config.js` => [`next.config.mjs`](https://nextjs.org/docs/pages/api-reference/next-config-js) and then we can use `import` and export default : 

```js
// @ts-check
import { withPlausibleProxy } from "next-plausible";

/** @type {import('next').NextConfig} */
const nextConfig = {
  ...
};

export default withPlausibleProxy()(nextConfig);
```

But yeah even ignoring is completely fine I feel

**Review by technophile-04 — APPROVED** [review]
Looks great to me 🙌!

File: packages/nextjs/next.config.js (hunk 1 above)
Comment (carletex) [reply packages/nextjs/next.config.js]: Changed to `.mjs` to use ESM :)

Thanks!

https://github.com/BuidlGuidl/abi.ninja/pull/47/commits/ba264a6f9b2830e2b565cea8dd9de2366ba5b9dc

**Review by portdeveloper — APPROVED** [review]
Looks good to me as well!

**Review by technophile-04 — APPROVED** [review]
Just fixed the conflicts [aaeb0e4](https://github.com/BuidlGuidl/abi.ninja/pull/47/commits/aaeb0e419289bb388a1766fd566ce364abe38a71)

@carletex maybe could check if going to https://abi-ninja-v2-1eh04vaxb-buidlguidldao.vercel.app/ increases the count for abi.ninja on plausible ideally it should increase right or lol I am missing something ? 

If it does we can merge 🙌

**Review by Pabl0cks — APPROVED** [review]
Tested in a fresh vercel and LGTM!!

**Pull request merged**